src/ui/user/UserDetailsModal.js

```diff
@@ -58,6 +65,12 @@ export default class UserDetailsModal extends Component {
     }
   }
 
+  componentDidUpdate() {
+    if (this.state.tab === 'rankGraph' && !this.state.graph) {
+      this._createGraph();
```

Comment:
A cleaner approach would be to create a separate RankGraph component that deals with all the lazy loading, so the user modal doesn't need to know anything about it.